data/coalition missions.txt

```diff
@@ -126,3 +126,14 @@ mission "Coalition: Contributor"
 			`	"For the sake of peace it is," says the Saryd. "The Heliarch military, well trained to use weapons only for defense and only for justice, they are. And to place the needs of the Coalition over the ambitions of their own native species, each one committed is. No such discipline do civilians have."`
 			label end
 			`	Apparently nothing is for sale here, but they assure you that many Coalition worlds will now grant you access to their shipyards and outfitters.`
+
+mission "Coalition License"
```

Comment:
I'd put "Patch" in the name and add a comment above that this is for pre-0.9.7 compatibility.